### PR TITLE
Activity (page + home)

### PR DESCRIPTION
**Pull request by carletex**
image

- Home: new activity section 
- Used the same styles for the /activity page (with diff card bg, because of the gradient for dark mode)
- The design differs a bit from Figma (talked with Andrea about it)

### CODE REVIEW

File: packages/react-app/components/EventRow.jsx [hunk 1]
```diff
@@ -16,20 +17,26 @@ const EventRow = ({ event }) => {
...
-      <Td whiteSpace="nowrap">
-        <DateWithTooltip timestamp={event.timestamp} />
-      </Td>
-      <Td>{eventDisplay(event)}</Td>
-    </Tr>
+    <Box backgroundColor={baseBlueColor} mb={4} p={8}>
+      <HStack spacing={8} align="flex-start" justify="space-between">
+        <HStack spacing={2.5} align="flex-start">
+          <NextLink href={`/builders/${userAddress}`} passHref>
+            <Link pos="relative" mt="-5.5px">
```
Comment (carletex) [inline packages/react-app/components/EventRow.jsx:28]: Not proud of this... haha But not sure about a better way of aligning this

File: packages/react-app/pages/index.jsx [hunk 2]
```diff
@@ -70,7 +75,9 @@ export async function getStaticProps() {
         buildsIncrementMonth: stats?.buildsIncrementMonth,
         streamedEthIncrementMonth: stats?.streamedEthIncrementMonth,
       },
+      events,
     },
+    // ToDo. Maybe a 15 min refresh? or load events in the frontend?
```
Comment (carletex) [inline packages/react-app/pages/index.jsx:80]: NO need to fix in this PR, but we have to think if:

1. We decrease this number to 15/30 min
2. We leave it as it's, but we fetch the activity from the frontend.

Not sure what's best (it depends on what we want to optimize)

File: packages/react-app/helpers/events.js
Comment (technophile-04) [inline packages/react-app/helpers/events.js:1]: Love the trick of converting it to array of title and desc !

File: packages/react-app/components/EventRow.jsx (hunk 1 above)
Comment (technophile-04) [reply packages/react-app/components/EventRow.jsx:29]: Lol, I am not sure either how would we handle this without changing Address component 

But I think its good and works nicely on mobile too !

File: packages/react-app/pages/index.jsx (hunk 2 above)
Comment (technophile-04) [reply packages/react-app/pages/index.jsx:80]: > Not sure what's best (it depends on what we want to optimize) 

Yeah, btw 15 / 30 min doesn't seem too low ? like if we keep it 15 / 30 min maybe its better we fetch it on frontend itself so we always show latest data to user ? Assuming that app.bg.com is visited less 

Maybe we decrease it from 6hr to 3hr ? 

Lol just thinking out loud, and really don't good idea / context how frequent bg events are emitted and totally agree that what we want to optimize !

**Review by technophile-04 — APPROVED** [review]
Tysm Carlos, looking great !!

File: packages/react-app/pages/index.jsx (hunk 2 above)
Comment (carletex) [reply packages/react-app/pages/index.jsx:80]: > Yeah, btw 15 / 30 min doesn't seem too low

Might be! 

If we don't cache anything, we are going to make a bunch of server/database requests on every visit to the homepage. I was thinking that even if we have 30 min as a caching window, we could save some.

I was trying to have a small number so the activity shows up to date.

But I think you are right, maybe we can start with 2/3 hours, and see.

We can always cache everything 6 hours, and just have the activity fresh, fetching from the frontend

Thanks Shiv!

**Pull request merged**